### ISSUE TITLE
Bumping targetSdkVersion to 28 breaks rotation support

### ISSUE THREAD

**Issue by moezbhatti** [bluelinelabs/Conductor#457]
If you set the targetSdkVersion to 28 and rotate the device, the controller won't be displayed once the activity is recreated

This behavior can be reproduced with the current version of the Demo app

**Comment by nksaroj** [comment]
Same here. I am also experiencing same issue. It's quite urgent. @EricKuck Appreciate your help as it's quite urgent.

**Comment by EricKuck** [comment]
Thanks for the report. The latest snapshot should be good to go now (as soon as it's done deploying).

**Comment by moezbhatti** [comment]
Thanks so much!

**Comment by nksaroj** [comment]
This issue is

**Comment by nksaroj** [comment]
Thanks @EricKuck  for looking into it. The issue still exist when the activity is destroyed.
You can simulate the issue by doing 

1. Open your device settings
2. Navigate to Settings --> System --> Developer options --> Apps section
3. Enable "Don't keep activities (Destroy every activity as soon as user leaves it)
4. Open the Conductor Demo that is targeted to API 28.
5. Click on the home  screen
6. Resume the app from the home screen
7. Blank screen is shown.

Please let me know if you are having any difficulties in replicating.

**Comment by inorichi** [comment]
2475436 also finally fixed #367. The issue was closed alongside a similar variant that happened when changing the backstack, but I could still reproduce with rotations + screen off/on.

It'd be nice to have a 2.1.5 release since a few annoying issues have been fixed.

**Comment by EricKuck** [comment]
Looks like the cause of this issue was that the Activity lifecycle call order changed (AGAIN) in P. From the docs:

```
For applications targeting platforms starting with Build.VERSION_CODES.P onSaveInstanceState(Bundle) will always be called after onStop(), so an application may safely perform fragment transactions in onStop() and will be able to save persistent state later.
```

This should now be completely fixed in 3b67822b25ac7da53a001cd8cbdc4a2c0da2184f.

**Comment by nksaroj** [comment]
@inorichi completely agree that develop should now merge to master  as 2.1.5. As we are using the app in the production I had to manually checkin the package to the source code rather pulling it from  the package manager to manage the risk of not introducing any unknown bugs due to the nature of develop branch.
